presto-main/src/main/java/com/facebook/presto/operator/SimplePagesHashStrategy.java

```diff
@@ -209,4 +210,10 @@ public boolean isPositionNull(int blockIndex, int blockPosition)
         }
         return false;
     }
+
+    @Override
+    public int compare(int leftBlockIndex, int leftBlockPosition, int rightBlockIndex, int rightBlockPosition)
+    {
+        throw new NotImplementedException();
```

Comment:
you were able to write compiler code, so it should be easy to write regular code, shouldn't it?